docs/proposals/formatted-output.md

```diff
@@ -332,57 +334,234 @@ $REGISTRY/$REPO@sha256:a3785f78ab8547ae2710c89e627783cfa7ee7824d3468cae6835c9f4e
+          "mediaType": "application/vnd.oci.image.manifest.v1+json",
+          "digest": "sha256:f520330e9f43c05859c532e67a25c9c765b144782ae7b872656192c27fd4e2dd",
+          "size": 1080,
+          "annotations": {
+            "io.cncf.notary.x509chain.thumbprint#S256": "[430ecf0685f8018443f8418f5d7134b146f28862116114925713635d5703fb69,9b1894f223d934cbd6575af3c6e1f6096b9221a7da132185f5a5cdc92235b5dc,23ffe2b8bdb9a1711515d4cffda04bc7f793d513c76c243f1020507d8669b7db]",
+            "org.opencontainers.image.created": "2024-03-15T22:54:42Z"
+          },
+          "artifactType": "application/vnd.cncf.notary.signature"
+        }
+      ]
     },
     {
-      "reference": "$REGISTRY/$REPO@sha256:50fd0dc107d84b5e7b402688000a7ed3aaf8a2692d5cb74da5277fa3c4cecf15",
+      "reference": "localhost:5000/kubernetes/kubectl@sha256:a811606b09341bab4bbc0a4deb2c0cb709ec9702635cbe2d36b77d58359ec046",
       "mediaType": "application/vnd.oci.image.manifest.v1+json",
-      "digest": "sha256:50fd0dc107d84b5e7b402688000a7ed3aaf8a2692d5cb74da5277fa3c4cecf15",
-      "size": 739,
+      "digest": "sha256:a811606b09341bab4bbc0a4deb2c0cb709ec9702635cbe2d36b77d58359ec046",
+      "size": 747,
       "annotations": {
-        "org.opencontainers.image.created": "2024-01-01T07:57:10Z"
+        "org.opencontainers.image.created": "2024-01-18T18:12:41Z"
       },
-      "artifactType": "sbom/example"
+      "artifactType": "application/vnd.in-toto+json",
+      "referrers": [
+        {
+          "reference": "localhost:5000/kubernetes/kubectl@sha256:04723fd7d00df77c6f226b907667396554bf9418dc48a7a04feb5ff24aa0b9ec",
+          "mediaType": "application/vnd.oci.image.manifest.v1+json",
+          "digest": "sha256:04723fd7d00df77c6f226b907667396554bf9418dc48a7a04feb5ff24aa0b9ec",
+          "size": 1080,
+          "annotations": {
+            "io.cncf.notary.x509chain.thumbprint#S256": "[430ecf0685f8018443f8418f5d7134b146f28862116114925713635d5703fb69,9b1894f223d934cbd6575af3c6e1f6096b9221a7da132185f5a5cdc92235b5dc,23ffe2b8bdb9a1711515d4cffda04bc7f793d513c76c243f1020507d8669b7db]",
+            "org.opencontainers.image.created": "2024-01-18T18:20:09Z"
+          },
+          "artifactType": "application/vnd.cncf.notary.signature"
+        }
+      ]
     }
   ]
-}
 ```
 
-> [!NOTE]
-> The `--format` flag will replace the existing `--output` flag. The `--output` will be marked as "deprecated" in ORAS v1.2.0 and will be removed in the future releases.
+#### Table format of `oras discover` output
+
+The `oras discover --format table` command currently lists only direct referrers in a table format. However, the table view has limitations—it does not effectively represent the hierarchical artifact reference relationships of all referrers. Users can't differentiate the reference relationship between each referrer in a table format output. See current sample output as follows:
```

Comment:
nit: "however" might be confusing here, as "The `oras discover --format table` command currently lists only direct referrers in a table format" implies that it cannot represent the hierarchical information.
How about:

```suggestion
The `oras discover --format table` command currently lists only direct referrers in a table format.  This table view has limitations: it does not effectively represent the hierarchical artifact reference relationships of all referrers. Users can't differentiate the reference relationship between each referrer in a table format output. See current sample output as follows:
```